### ISSUE TITLE
Segmentation fault on draw_geometries

### ISSUE THREAD

**Issue by scimad** [intel-isl/Open3D#1941]
**Bug**
On v10.0, I get segmentation fault at the end of the program when the visualization window is loaded. Oddly enough, this happens only when `set_verbosity_level()` is set to `Debug` but I don't get the `segmentation fault` not when set to `Error` or `set_verbosity_level()` is removed altogether.

**To Reproduce**
```
import open3d as o3d

o3d.utility.set_verbosity_level(o3d.utility.VerbosityLevel.Debug)
mesh_frame = o3d.geometry.TriangleMesh.create_coordinate_frame(size=0.6)
o3d.visualization.draw_geometries([mesh_frame])

print ("done")
```
**Expected behavior**
Should end without segmentation fault.

**Environment:**

- Operating system: Ubuntu 20.04
- Python version: Python 3.8
- Open3D version: v0.10.0
- Remote workstation: no
- Installed via pip3 (also same issue persists when compiled from source as well)
- Compiler version (while built from source): gcc version 9.3.0

**Comment by AlphaPav** [comment]
did you solve this problem? thanks!

**Comment by yxlao** [comment]
Fixed in #2807. Will be included in the December 2020 v0.13 release.
